.github/workflows/smoke-test-daily.yml

```diff
@@ -52,3 +52,103 @@ jobs:
           pnpx wc-e2e test:e2e plugins/woocommerce/tests/e2e/specs/smoke-tests/update-woocommerce.js
           pnpx wc-e2e test:e2e
           pnpx wc-api-tests test api
+
+  build:
+    name: Build zip for PR
+    runs-on: ubuntu-latest
+    steps:
+        - name: Checkout code
+          uses: actions/checkout@v2
+
+        - name: Build
+          id: build
+          uses: woocommerce/action-build@trunk
+          env:
+              BUILD_ENV: e2e
+
+        - name: Upload PR zip
+          uses: actions/upload-artifact@v2
+          env:
+              GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          with:
+              name: woocommerce
+              path: ${{ steps.build.outputs.zip_path }}
+              retention-days: 7
+    
+  test-plugins:
+    name: Smoke tests with ${{ matrix.plugin }} plugin installed
+    runs-on: ubuntu-18.04
+    needs: [build]
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - plugin: 'WooCommerce Payments'
+            repo: 'automattic/woocommerce-payments'
+          - plugin: 'Facebook for WooCommerce'
```

Comment:
We will need to remove this plugin until the debug warning is fixed because it breaks the dashboard.

Reply:
Good call--it's nice it already caught our first bug!

Will swap this one out for https://github.com/woocommerce/woocommerce-paypal-payments

Reply:
This has been pushed up: https://github.com/woocommerce/woocommerce/pull/31400/commits/07a96e3aaa29e877e8e3812923f490a7cc7eba24